package.json

```diff
@@ -22,8 +23,10 @@
     "@lokalise/eslint-config-lokalise-frontend": "0.3.0",
     "eslint-config-prettier": "8.6.0",
     "eslint-plugin-prettier": "4.2.1",
-    "jest": "29.4.2",
     "prettier": "2.8.4",
-    "vite": "4.1.1"
+    "ts-jest": "29.0.5",
```

Comment:
probably not needed anymore

Reply:
Hmm turns out the jest lint rules still require this to infer the jest version to test against.

But this is all a bit odd given that we are not using jest...

There is a vitest specific eslint-plugin however it has far fewer rules than the jest plugins we use... and the jest plugins are still compatible with the test syntax we use (i.e. vitest doesn't write the tests differently... so all the same lint rules can apply).

Reply:
it's not possible to set jest version explicitly somewhere?

Reply:
Turns out I didn't need to. I have to assume it is somehow inferred from the vitest version.